### PR TITLE
Update of locales fr, es.

### PR DESCRIPTION
**Pull request by Bundy01**

### CODE REVIEW

**Comment by actionless** [comment]
thanks!

File: locale/es.po [hunk 1]
```diff
@@ -473,9 +473,9 @@ msgstr "[a] abortar la compilación de todos los paquetes"
 msgid "[c] checksums skip"
 msgstr "[c] omisión de sumas de comprobación"
 
-#: pikaur/install_cli.py:463
+#: pikaur/install_cli.py:464
 msgid "[c] git checkout -- '*'"
-msgstr "[c] git checkout -- '*'"
+msgstr "[c] extracción git -- '*'"
```
Comment (actionless) [inline locale/es.po]: already after merging found this problem (and the same in `fr.po`)

(unless with es/fr locale you actually could ask `git` to make some extrazione for you)

**Comment by Bundy01** [comment]
I saw that on the other locales "git checkout" was not translated. Do I have to do a PR again to put it back in fr.po and es.po?

**Comment by actionless** [comment]
yup, thanks in advance!

**Pull request merged**